backend/docs/devices_v1_deployments.oai.yaml

```diff
@@ -0,0 +1,396 @@
+openapi: 3.0.1
+info:
+  description: |
+    An API for device firmware deployments. Intended for use by devices.
+
+    Devices can get new updates and send information about current deployment status.
+  title: Deployments
+  version: "1"
+servers:
+  - url: https://hosted.mender.io/api/devices/v1/deployments
+paths:
+  /device/deployments/next:
+    get:
+      description: |
+        On success, either an empty response or a DeploymentInstructions object
+        is returned depending on whether there are any pending updates.
+      operationId: Check Update
+      parameters:
+        - description: currently installed artifact
+          in: query
+          name: artifact_name
+          required: true
+          schema:
+            type: string
+        - description: Device type of device
+          in: query
+          name: device_type
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          content:
+            application/json:
+              example:
+                id: w81s4fae-7dec-11d0-a765-00a0c91e6bf6
+                artifact:
+                  artifact_name: my-app-0.1
+                  source:
+                    uri: https://aws.myupdatebucket.com/image123
+                    expire: 2016-03-11T13:03:17.063493443Z
+                  device_types_compatible:
+                    - rspi
+                    - rspi2
+                    - rspi0
+              schema:
+                $ref: "#/components/schemas/DeploymentInstructions"
+          description: Successful response.
+        "204":
+          content: {}
```

Comment:
is this one on purpose explicitly mentioned as empty?

Reply:
It's just an artifact of the generated output. I have not touched any of these files apart from removing some internally generated fields.